infrastructure/alert_action_group.tf

```diff
@@ -0,0 +1,26 @@
+data "azurerm_key_vault_secret" "source_bsp_email_secret" {
+  name      = "send-letter-alert-email"
+  vault_uri = "${module.send-letter-key-vault.key_vault_uri}"
+}
```

Comment:
Action group has been removed some time ago and secret doesn't exist in key vault anymore. Can you set up again those? I can DM what values we have for bulkscan